firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandler.java

```diff
@@ -189,21 +189,32 @@ public ConfigFetchHandler(
                   createThrottledMessage(backoffEndTime.getTime() - currentTime.getTime()),
                   backoffEndTime.getTime()));
     } else {
-      Task<InstanceIdResult> instanceIdTask = firebaseInstanceId.getInstanceId();
+      Task<String> installationIdTask = firebaseInstallations.getId();
+      Task<InstallationTokenResult> installationTokenTask = firebaseInstallations.getToken(false);
```

Comment:
+1 for variable naming :)
I would recommend to have it consistent with variables above to not confuse readers.
`installationTokenTask` sounds good.
Another option is `fisAuthTokenTask` or `installationAuthTokenTask`, but I think all options are fine.

Reply:
Yeah, sounds good. I like the specificity.